src/main/java/com/github/jcustenborder/kafka/connect/redis/RedisConnectorConfig.java

```diff
@@ -153,7 +165,7 @@ public static ConfigDef config() {
         ).define(
             ConfigKeyBuilder.of(DATABASE_CONFIG, ConfigDef.Type.INT)
                 .documentation(DATABASE_DOC)
-                .defaultValue(1)
+                .defaultValue(0)
```

Comment:
What was this change for?

Reply:
Typically redis defaults to database 0 when you connect so I set this default to be more in line with something someone familiar with redis would expect.